metricbeat/tests/system/test_config.py

```diff
@@ -65,5 +65,23 @@ def test_compare_config(self):
 
         assert same == True
 
+    def test_invalid_config_with_removed_settings(self):
+        """
+        Checks if metricbeat fails to load a module if remove settings have been used:
+        """
+        self.render_config_template(modules=[{
+            "name": "system",
+            "metricsets": ["cpu"],
+            "period": "5s",
+        }])
+
+        exit_code = self.run_beat(extra_args=[
+            "-E",
+            "metricbeat.modules.0.filters.0.include_fields='field1,field2'"
+        ])
+        assert exit_code == 1
+        assert self.log_contains("setting 'metricbeat.modules.0.filters'"
```

Comment:
Since `filters` is now renamed, can you update the default config files [here](https://github.com/elastic/beats/blob/master/metricbeat/modules.d/system.yml#L23) and [here](https://github.com/elastic/beats/blob/master/metricbeat/modules.d/redis.yml.disabled#L24).